src/renderer/api/endpoints/events.api.ts

```diff
@@ -75,6 +75,10 @@ export class KubeEvent extends KubeObject {
 
     return formatDuration(diff, true);
   }
+
+  getLastSeenTimeDiffFromNow(): number {
```

Comment:
Just a note that these kind of helper methods should not be in the api itself.. but we already have similar method so maybe it's fine to add "one more".

Reply:
Okay makes sense, we probably want to remove both in the future. Shall I not make it worse in this PR?